### ISSUE TITLE
Not able to run karate tests via VSCode extension on MacOS

### ISSUE THREAD

**Issue by DrAlabala** [karatelabs/vscode-extension#21]
I have just updated to the latests version of the extension: 2.0.3
Also installed the extension:  https://marketplace.visualstudio.com/items?itemName=redhat.java as per the documentation

Now when I try to Run >> a scenario  
<img width="157" alt="image" src="https://github.com/karatelabs/vscode-extension/assets/3792304/70c64a17-3e35-4f8b-a9bd-da90aa44d7d4">
Then in the Output i see the following message

`Error: Could not find or load main class com.intuit.karate.Main
Caused by: java.lang.ClassNotFoundException: com.intuit.karate.Main` 
thrown by Karete Run

And from the Karate Log I can only see 
`2024-03-15 17:41:45.766 [info] process disposed: ["Karate Run"]
2024-03-15 17:41:45.766 [info] process spawn: ["Karate Run",65661]
2024-03-15 17:41:45.804 [info] process close: ["Karate Run",65661,1]`

I have check my Java setup and everything seems to be fine!

What might be the reason for not able to find or load the `main class com.intuit.karate.Main`

Moving back to Karate VSCode extension version v1.4.1 and again everything runs smoothly

**Comment by ptrthomas** [comment]
@DrAlabala can you follow the instructions here and change the log level to `Debug`: https://github.com/karatelabs/vscode-extension#troubleshooting - then please paste the log, which should have more details. would you also be okay to get on a video call on Monday so we can look at this?

**Comment by DrAlabala** [comment]
Hi @ptrthomas and thanks for the reply.
I should had to clarify that our current project is not a java based project created with `maven` or `gradle`. 
When we started with karate VSCode extension for our needs, we just created a desired folder structure, we defined our `.feature` files and feed them with lot's of different test data.  No additional extensions were installed in terms of supporting Java.
As I mentioned above the latest version that we were using and where everything was running nice and smooth was version `v1.4.1`. For it I can see that the configuration for Run:Mode is `bundled` - the build-in  Karate runtime.

Now, with the latest version of karate extension, It seems that it is not the case anymore.
I have tried with using the standalone .jar and applied the need changes into the extensions settings, and I was able to run the our tests.
